### DISCUSSION TITLE
Is there a way to change username, or better simply disable admin login?

### DISCUSSION THREAD

**Discussion by sao321** [owncast/owncast#3090] [Q&A]
Hi,

I'd like to improve security against brute force. The silly way would be changing username, but I can't find a way to do this.

So what I have done is that I added a NGINX Basic Authentication for /admin directory, together with fail2ban. However, this resulted an infinite loop, probably a conflict with the native authentication, and the admin panel can't even be loaded, so I'm wondering if I can disable admin login all together and letting NGINX Basic Authentication to handle everything.

Just wondering if anyone have any good solution? Thanks!

**Comment by gabek** [comment]
There is no way to do this.

You might find this thread useful, where somebody else added additional HTTP Basic auth and there's discussion about how to do so. https://github.com/owncast/owncast/issues/2653

**Reply by sao321** [reply]
Unfortunately my site is nginx based, it will be a huge pain to add another layer, but thank you very much for your reply.

**Reply by sao321** [reply]
Actually, thank you very much for the link. After reconsidering and looking at 4censord's solution, an idea came to mind: What if I use the same login and password for both nginx authentication and native authentication? It turns out that it works, but since nginx authentication is protected by fail2ban, it is still safeguarded against brute force attacks. It's not an ideal solution, but it's better than nothing.

**Comment by ozgurkazancci** [comment]
You might wish to check: https://github.com/owncast/owncast/issues/3625
